File: C/soma_vetor.c
#include <stdio.h>
#include <locale.h>  // Inclusão da biblioteca de localização

int main(){
    setlocale(LC_ALL, "");  // Configura as opções de localização

    int n;  // Variável para armazenar a quantidade de números que serão digitados
    double soma, media;  // Variáveis para armazenar a soma e a média dos números

    // Solicita ao usuário a quantidade de números que serão digitados
    printf("Quantos números você vai digitar? ");
    scanf("%d", &n);

    double vet[n];  // Declaração do vetor para armazenar os números
    soma = 0.0;  // Inicialização da variável soma com 0.0

    // Loop para ler os números do usuário e somá-los
    for (int i = 0; i < n; i++){
        printf("Digite um numero: ");
        scanf("%lf", &vet[i]);
        soma = soma + vet[i];
    }

    // Exibição dos números digitados
    printf("\nVALORES = ");
    for (int i = 0; i < n; i++){
        printf("%.1lf ", vet[i]);
    }

    // Exibição da soma dos números
    printf("\nSOMA = %.2lf\n", soma);

    // Cálculo e exibição da média
    media = soma / n;
    printf("MEDIA = %.2lf\n", media);

    return 0;
}

File: Java/soma_vetor.java
import java.util.Locale;
import java.util.Scanner;

public class soma_vetor {

    public static void main(String[] args) {

        Locale.setDefault(Locale.US);
        Scanner sc = new Scanner(System.in);

        System.out.print("Quantos números você vai digitar? ");
        int n = sc.nextInt();

        double[] vet = new double[n];

        for (int i = 0; i < n; i++){
            System.out.print("Digite um número: ");
            vet[i] = sc.nextDouble();
        }

        System.out.print("VALORES = ");
        for (int i = 0; i < n; i++){
            System.out.print(String.format("%.1f ", vet[i]));
        }
        System.out.println(" ");

        double soma = 0;

        for (int i = 0; i < n; i++){
            soma += vet[i];
        }

        System.out.println("SOMA = " + String.format("%.2f", soma));

        double media = soma / n;
        System.out.println("MEDIA = " + String.format("%.2f", media));

        sc.close();

    }
}
